src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -215,17 +216,18 @@ private void GroupFixes(Workspace workspace, IEnumerable<CodeFixCollection> fixC
                     var fixes = fixCollection.Fixes;
                     var fixCount = fixes.Length;
 
+                    Func<CodeAction, SuggestedActionSet> getFixAllSuggestedActionSet = codeAction =>
+                                CodeFixSuggestedAction.GetFixAllSuggestedActionSet(codeAction, fixCount, fixCollection.FixAllContext,
+                                    workspace, _subjectBuffer, _owner._editHandler);
+
```

Comment:
In the case of SuppressionCodeAction, shouldn't we call CodeFixSuggestedAction.GetFixAllSuggestedActionSet() for each of the individual 'nested' actions inside the SuppressionCodeAction instead of for the outer SuppressionCodeAction itself?